src/main/java/nextstep/subway/line/domain/Sections.java

```diff
@@ -49,6 +57,15 @@ public void addSection(Section requestSection) {
         addSectionInExistSections(requestSection);
     }
 
+    public void deleteSection(Station deletedStation) {
+        List<Section> sectionsContainDeletedStation = findSectionsContainDeleteStation(deletedStation);
```

Comment:
```suggestion
        List<Section> sections = findSectionsBy(station);
```

실제 findSectionsContainDeleteStation 메서드가 하는 일은 station으로 section을 찾는 기능이네요.
메서드명을 잘못 쓰면 코드 파악이 어려워집니다.

과정들이 합쳐져서 delete하는 것이니 각 과정은 간략하게 이름을 붙여주세요.